src/OfficeService.Validation/Office/CreateOfficeRequestValidator.cs

```diff
@@ -8,11 +8,20 @@ public class CreateOfficeRequestValidator : AbstractValidator<CreateOfficeReques
   {
     public CreateOfficeRequestValidator()
     {
+      RuleFor(request => request.Name)
+        .NotEmpty().WithMessage("Name must not be empty.");
+
       RuleFor(request => request.City)
-        .Must(c => !string.IsNullOrEmpty(c?.Trim())).WithMessage("City must not be empty.");
+        .NotEmpty().WithMessage("City must not be empty.");
 
       RuleFor(request => request.Address)
-        .Must(a => !string.IsNullOrEmpty(a?.Trim())).WithMessage("Address must not be empty.");
+        .NotEmpty().WithMessage("Address must not be empty.");
+
+      RuleFor(request => request.Latitude)
+        .NotEmpty().WithMessage("Latitude must not be empty.");
```

Comment:
Latitude and Longitude can't be empty or null as double, so NotEmpty() here means "!= default value for double". If this is what you meant then ok.